README.md

```diff
@@ -1,8 +1,31 @@
 # CSS Learning Resources
+
 A collection of the best CSS resources. 
 
+## Websites for CSS
+
+- https://www.w3schools.com/css/
+- https://developer.mozilla.org/en-US/docs/Web/CSS
+- https://www.tutorialspoint.com/css/index.htm
+- https://web.dev/learn/css/
+- https://www.javatpoint.com/css-tutorial
+
+
+
 ## Courses
+
 * [CSS for JavaScript Developers](https://css-for-js.dev/) by @ui_natalie
+* https://www.udemy.com/course/html5-fundamentals-for-beginners/ by Stone RIver
+* https://www.udemy.com/course/css-flexbox-basic-to-advanced-with-responsive-project/ by Abdullah Nahian
+* https://www.udemy.com/course/web-development-learn-by-doing-html5-css3-from-scratch-introductory/  by  [Bradley Berger]
+* https://www.freecodecamp.org/news/get-started-with-css-in-5-minutes-e0804813fc3e/
+* https://www.udemy.com/course/css-the-complete-guide-incl-flexbox-grid-sass/?ranMID=39197&ranEAID=JVFxdTr9V80&ranSiteID=JVFxdTr9V80-.WqxzIGm4.J8afJ.HVu5tw&LSNPUBID=JVFxdTr9V80&utm_source=aff-campaign&utm_medium=udemyads
+* https://www.coursera.org/learn/responsivedesign?ranMID=40328&ranEAID=JVFxdTr9V80&ranSiteID=JVFxdTr9V80-N8bfvHnLZKmwhdi2fKThTA&siteID=JVFxdTr9V80-N8bfvHnLZKmwhdi2fKThTA&utm_content=10&utm_medium=partners&utm_source=linkshare&utm_campaign=JVFxdTr9V80
+* https://www.edx.org/learn/css
+* https://www.codecademy.com/catalog/language/html-css?utm_source=pepperjam&utm_medium=affiliate&utm_term=231808&clickId=3835390508&pj_creativeid=8-12462&pj_publisherid=231808
 
 ## News
+
 * [CSS Tricks](https://css-tricks.com/) by @ui_natalie
+* https://www.feedspot.com/infiniterss.php?_src=feed_title&followfeedid=124822&q=site:https%3A%2F%2Fstackoverflow.com%2Ffeeds%2Ftag%3Ftagnames%3Dcss
+* http://css-tricks.com/
```

Comment:
This is a duplicate to the first line under News with CSS Tricks.

Reply:
I ll correct it